src/ansys/mapdl/core/convert.py

```diff
@@ -586,36 +608,40 @@ def translate_line(self, line):
         # Cleaning ending empty arguments.
         # Because of an extra comma added to toffst command when generating ds.dat.
         line_ = line.split(",")[::-1]  # inverting order
+
         for ind, each in enumerate(line_):
-            if each:
+            if each.strip():  # strip to remove spaces in empty arguments
                 break
-            else:
-                line_.pop(ind)
-        line = ",".join(line_[::-1])
+
+        line = ",".join(line_[ind:][::-1])
 
         # remove trailing comma
         line = line[:-1] if line[-1] == "," else line
+        LINE = line.upper()
 
-        cmd_ = line.split(",")[0].upper()
+        CMD_ = line.split(",")[0].upper()
+        CMD4_ = CMD_[:4]
 
-        if cmd_[:4] in ["SOLV", "LSSO"] and self._comment_solve:
+        items = line.split(",")
+
+        if CMD4_ in ["SOLV", "LSSO"] and self._comment_solve:
```

Comment:
Style wise, I think we should not use ALL CAPS unless it's a module level global. Simply `cmd` or `cmd4` works. Trailing underscore doesn't seem necessary either.

Reply:
Mmmmhhh... I did all caps in some variables because the stored string is in capital, so I thought it was an easy way to express that. But you are right. I will change it.